roles/pulp-webserver/templates/nginx.conf.j2

```diff
@@ -75,7 +75,8 @@ http {
             proxy_pass http://pulp-api;
         }
 
-        include conf.d/*.conf;
+#        
+#        include conf.d/*.conf;
```

Comment:
@aspoliveira1981 `include conf.d/*.conf;` cannot be removed, as it is used for customizing plugin URLs:
https://docs.pulpproject.org/plugins/plugin-writer/concepts/index.html#configuring-reverse-proxy-with-custom-urls

Reply:
thanks @fao89 but I wonder about the other conf files you mentioned:

[root@ip-10-2-1-88 templates]# cd /etc/nginx/conf.d/
[root@ip-10-2-1-88 conf.d]# ls
default.conf
[root@ip-10-2-1-88 conf.d]# 

From this installation I just got the default.conf, maybe we need to add under the templates/ the other jinja files.

Reply:
@aspoliveira1981 can you `tree /etc/nginx/` so we can more completely see the state of your system?